mcstatus/tests/test_deprecated_decorator.py

```diff
@@ -46,11 +46,13 @@ def test_deprecated_class_return_result():
 
 def test_deprecated_function_with_methods():
     with pytest.raises(ValueError):
-        deprecated(methods=["__str__"])(lambda: None)
+        deprecated(methods=["__str__"])(lambda: None)  # pragma: no branch
 
 
 def test_deprecated_class_without_methods():
-    test_cls: Any = type("TestCls", (), {"foo": lambda x: x})  # https://github.com/microsoft/pyright/discussions/3438
+    test_cls: Any = type(  # pragma: no branch
+        "TestCls", (), {"foo": lambda x: x}
+    )  # https://github.com/microsoft/pyright/discussions/3438
```

Comment:
```suggestion
    # https://github.com/microsoft/pyright/discussions/3438
    test_cls: Any = type("TestCls", (), {"foo": lambda x: x})  # pragma: no branch
```

Reply:
Fixed in [the last force-push](https://github.com/py-mine/mcstatus/compare/010c1e1bb95f8a022a42b24c1b5b2f5b39e6d2cb..20cad3b7031cbef5dd4ccc91d0eaf6987a4f6682).